.github/workflows/ci.yaml

```diff
@@ -20,10 +20,49 @@ env:
   IMAGE_NAME: "ramen"
   IMAGE_TAG: "ci"
   DOCKERCMD: "podman"
+  OPERATOR_SUGGESTED_NAMESPACE: "ramen-system"
 defaults:
   run:
     shell: bash
 jobs:
+  definevars:
```

Comment:
Was unable to define vars in the env section above, based on github repository [variables](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows), hence this added section for output processing in other jobs.